### ISSUE TITLE
Problem during Installation

### ISSUE THREAD

**Issue by guruvamsi-policharla** [BBN-Q/MathieuFunctions.jl#6]
The MathieuFunctions package doesn't seem to show up in the Julia package manager. 
On trying to add MathieuFunctions in the REPL it shows an error message saying

"ERROR: The following package names could not be resolved:
 \* MathieuFunctions (not found in project, manifest or registry)
Please specify by known `name=uuid`."

The version of Julia is 1.1.1.

**Comment by matthewware** [comment]
This package isn't registered yet.  It's on my to-do list but unlikely to happen in the next few days.  you can still install with the usual:
```pkg> add "https://github.com/BBN-Q/MathieuFunction.jl"```

That said, the package is broken in Julia > 1.0.  This is also on my list of things to fix but not likely to happen ASAP.  Something broke in the update from v0.6 syntax to v1.0.  Ideally I'd like to fix the issues and then register the package.

**Comment by matthewware** [comment]
The package is working again.  That is, tests are at least passing.  I hazard to suggest you give things a try now.  Let me know how things go.

**Comment by matthewware** [comment]
Currently waiting on the package registrations to be merged into the registry.

**Comment by matthewware** [comment]
Package is now registered.  Closing this issue for now.  Let me know if you have any other issues.
